src/compas/geometry/primitives/polyline.py

```diff
@@ -340,7 +340,7 @@ def divide_polyline(self, num_segments):
 
         return self.divide_polyline_by_length(segment_length, False)
 
-    def divide_polyline_by_length(self, length, strict=True):
+    def divide_polyline_by_length(self, length, strict=True, tol=1e-06):
```

Comment:
Add `tol` to the parameters list in the docstring, please...

Reply:
oh, also, there shouldn't be blank lines between the parameters... it makes the docs look funny... see https://compas.dev/compas/latest/api/generated/compas.geometry.Polyline.divide_polyline_by_length.html#compas.geometry.Polyline.divide_polyline_by_length

Reply:
just fyi, in `offset_polyline` tol isn't referenced!  https://github.com/compas-dev/compas/blob/be558946aaf34b30dec0cdf4d2e52d805a449bf5/src/compas/geometry/offset/offset.py#L125


Will make the changes though!